packages/app/package.json

```diff
@@ -35,6 +35,7 @@
     "@roadiehq/backstage-plugin-jira": "^1.0.13",
     "@roadiehq/backstage-plugin-security-insights": "^1.2.11",
     "@roadiehq/backstage-plugin-travis-ci": "^1.0.12",
+    "@roadiehq/plugin-backstage-plugin-prometheus": "^0.1.0",
```

Comment:
Should be ^1.0.0 ? (the checks pointed this out to me)

Reply:
Bumped to be ready for release, didn't remember to update this :+1: